### PR TITLE
Add JHU Data as a Modeling Source [WIP]

### PR DESCRIPTION
**Pull request by davidstrauss**

### CODE REVIEW

File: libs/CovidDatasets.py [hunk 1]
```diff
@@ -83,82 +146,179 @@ def backfill(self, series, model_interval):
...
+	_POPULATION_URL = r'https://raw.githubusercontent.com/covid-projections/covid-data-model/master/data/populations.csv'
+	_BEDS_URL = r'https://raw.githubusercontent.com/covid-projections/covid-data-model/master/data/beds.csv'
+
+	def __init__(self):
+		super().__init__(start_date=datetime.datetime(year=2020, month=3, day=3))
+
+	def transform_jhu_timeseries(self):
+		""""Takes a list of JHU daily reports, mashes them into a single report, then restructures and renames the data
+		to fit the model's expectations"""
+		daily_reports_dir = r'data\jhu\csse_covid_19_daily_reports'
```
Comment (mikelehen) [inline libs/CovidDatasets.py]: These need to be `/` instead of `\` to work on unix/mac.

**Comment by jredding** [comment]
This is a huge PR that has changes all over the place. 

I get the data/jhu . folder 

but what are we accomplishing with all the files within /test/* ?

File: libs/CovidDatasets.py [hunk 2]
```diff
@@ -87,84 +151,185 @@ def backfill(self, series, model_interval):
...
+            columns={
+                'Country/Region': 'country',
+                'Province/State': 'state',
+                'County': 'county',
+                'Confirmed': 'cases',
+                'Deaths': 'deaths',
+                'Recovered': 'recovered'
+            }
+        )
+        return full_report.drop('index', axis=1)
```
Comment (mikelehen) [inline libs/CovidDatasets.py]: nit: `axis='columns'` would probably be more readable.

File: notebook_runtime.ipynb [hunk 3]
```diff
@@ -11,15 +11,17 @@
     "import datetime\n",
     "import seaborn as sb\n",
     "import time\n",
+    "import os.path\n",
     "from ipywidgets import interact, fixed\n",
     "from libs.CovidTimeseriesModel import CovidTimeseriesModel\n",
-    "from libs.CovidDatasets import CovidDatasets\n",
+    "from libs.CovidDatasets import CDSDataset, JHUDataset\n",
+    "from libs.CovidUtil import CovidUtil\n",
```
Comment (mikelehen) [inline notebook_runtime.ipynb]: is this used?  Should it be? (would it simplify the notebook?)

**Comment by mikelehen** [comment]
This is obsolete.  We moved to the ghu-new branch and merged that in #26 .

**Pull request closed without merging**